sources/api/early-boot-config/Cargo.toml

```diff
@@ -26,6 +27,10 @@ tokio = { version = "0.2", default-features = false, features = ["macros", "rt-t
 #tokio = { version = "0.3", default-features = false, features = ["macros", "rt-multi-thread"] }
 toml = "0.5"
 
+[target.'cfg(target_arch = "x86_64")'.dependencies]
+# vmw_backdoor includes x86_64 assembly, prevent it from building for ARM
+vmw_backdoor = "0.2"
```

Comment:
Are you planning to do #1404 before merging this?  It'd be nice not to have a known (and awkward) build failure, and I don't think this is blocking anything else yet..?

Reply:
#1431 is out to take care of #1404 and I plan to merge it before I merge this PR.